docs/user/openstack/README.md

```diff
@@ -44,6 +47,69 @@ enough to store the ignition config files, so they are served by swift instead.
 * You may need to increase the security group related quotas from their default
   values. For example (as an OpenStack admin) `openstack quota set --secgroups 100 --secgroup-rules 1000 <project>`
 
+
+* A file `install-config.yaml` is required. It defines your cluster topology. Save it because a failed installation will delete it. It looks like this:
+
+```
+apiVersion: v1
+baseDomain: example.com
+controlPlane:
+  name: master
+  platform:
+    openstack:
+      type: m1.master
```

Comment:
I'd prefer to remove the entire `controlPlane` section.

First, to my knowledge the only supported configuration is 3 masters.

Second, the flavor specified here is not used for the initial masters created by Terraform, only by the subsequent ones in the MachineSets. As such, it should be identical to the top-level `platform.openstack.flavorName` config and if you just leave `controlPlane.platform` out, that's exactly what happens.

If you want to leave the replicas option in, then please remove the flavor section at least.